reference/orders.v3.yml

```diff
@@ -661,13 +648,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/MetafieldResponse'
-        '404':
```

Comment:
I guess this one is possible to get, I'm just not sure about the response body :)

Reply:
@6juara9 I updated the response body.
![Screenshot 2024-05-03 at 11 44 04 AM](https://github.com/bigcommerce/docs/assets/64287776/c4c49772-3839-4b9f-a0fc-e2893b632507)

Reply:
also not sure if we want it to look like this:
![Screenshot 2024-05-06 at 14 00 19](https://github.com/bigcommerce/docs/assets/136105824/a92a0d4b-3d3b-4e42-ab89-30c109cff4c6)
The text is on response code description not title part of body. Wdyt?

Reply:
Hi @kskomro My preview looks like the following:
<img width="645" alt="Screenshot 2024-05-06 at 9 19 56 AM" src="https://github.com/bigcommerce/docs/assets/64287776/bba4dc2b-9e0e-4614-a66b-3ea6c571c3f3">